src/components/calcite-progress/calcite-progress.scss

```diff
@@ -1,30 +1,24 @@
 :host {
-  position: relative;
-  display: block;
-  width: 100%;
+  @apply relative block w-full;
 }
 
 .track,
 .bar {
-  position: absolute;
-  top: 0;
+  @apply absolute top-0;
   height: 2px;
 }
 
 .track {
-  background: var(--calcite-ui-border-3);
-  z-index: 0;
-  width: 100%;
-  overflow: hidden;
+  @apply z-0 w-full overflow-hidden;
+  background: theme("borderColor.color.3");
 }
 
 .bar {
-  background-color: var(--calcite-ui-brand);
-  z-index: 0;
+  @apply bg-brand z-0;
 }
 
 .indeterminate {
-  width: 20%;
+  width: theme("width.[1/5]");
```

Comment:
I think you can just use `w-1/5` as an `@apply`?

Reply:
Ah, forgot to remove the old value. thx!